docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -13,99 +13,208 @@ description: guide for doing a lift and shift migration of airflow to dagster
+In the `dagster_migration.py` file you created in [Step 1](#step-1-prepare-your-project-for-a-new-dagster-python-module), use <PyObject module="dagster_airflow" object="make_dagster_definitions_from_airflow_dags_path" /> and pass in the file path of your Airflow DagBag. Dagster will load the DagBag and convert all DAGs into Dagster jobs and schedules.
 
 ```python file=/integrations/airflow/migrate_repo.py
 import os
 
 from dagster_airflow import (
-    make_dagster_repo_from_airflow_dags_path,
+    make_dagster_definitions_from_airflow_dags_path,
 )
 
-migrated_airflow_repo = make_dagster_repo_from_airflow_dags_path(
-    os.path.join(os.environ["AIRFLOW_HOME"], "dags"),
-    "migrated_airflow_repo",
+migrated_airflow_definitions = make_dagster_definitions_from_airflow_dags_path(
+    os.path.abspath("./dags/"),
 )
 ```
 
-Under the hood, Dagster is running the exact operator code as you were in Airflow. You will be able to view your normal Airflow `stdout`/`stderr` logs as compute logs in Dagit.
+---
+
+## Step 4: Verify the DAGs are loading
+
+In this step, you'll spin up Dagit, Dagster's web-based UI, and verify that your migrated DAGs are loading. **Note**: Unless the migrated DAGs are very, very simple, it's unlikely they'll execute correctly at this point. That's okay - we'll fix it in a bit. Starting Dagit is the first step in our development loop, allowing you to make a local change, view it in Dagit, and debug any errors.
+
+1. Run the following to start Dagit:
+
+   ```bash
+   dagster dev -f ./migrate_repo.py
+   ```
+
+2. In your browser, navigate to <http://localhost:3001>. You should see a list of Dagster jobs that correspond to the DAGs in your Airflow DagBag.
+
+3. Run one of the simpler jobs, ideally one where you're familiar with the business logic. Note that it's likely to fail due to a configuration or permissions issue.
+
+4. Using logs to identify and making configuration changes to fix the cause of the failure.
+
+Repeat these steps as needed until the jobs run successfully.
+
+#### Containerized Operator Considerations
+
+There are a variety of Airflow Operator types that are used to create container runs in various container execution environments, for example kubernetes or amazon ECS, when getting things working locally we'd recommend trying to execute those containers locally whenever possible and only relying on remote execution when its either unrealistic or impossible to locally emulate the cloud environment. For doing local execution it likely means that you will need to have a local kubernets cluster running, we'd recommnd docker's built in k8s environment, and to be able to pull down the container images that will be needed for execution to your local machine.
```

Comment:
I did a pass for typos and added a last sentence about Cloud:

There are a variety of Airflow Operator types that are used to launch compute in various external execution environments, for example Kubernetes or Amazon ECS. When getting things working locally we'd recommend trying to execute those containers locally unless it's either unrealistic or impossible to emulate the cloud environment. For example is you use the K8sPodOperator, it likely means that you will need to have a local Kubernetes cluster running, and in that case we recommend docker's built-in Kubernetes environment. You also need to be able to pull down the container images that will be needed for execution to your local machine.

If local execution is impossible, we recommend using Branch Deployments in Dagster Cloud, which is a well-supported workflow for cloud-native development.

Reply:
^---- Re: this are we sure we want to recommend this? I highly doubt that most people's containers can work on a laptop, even if they get a local k8s cluster going. We might be leading people down the wrong path.